refreshVersions/src/main/kotlin/de/fayard/versions/internal/UsedDependenciesWriting.kt

```diff
@@ -30,16 +39,41 @@ internal fun Project.writeUsedDependencies() {
 
     file.bufferedWriter().use { writer ->
         allDependencies.forEach { dependency ->
-            writer.append(dependency.group)
-            writer.append(':')
-            writer.append(dependency.name)
-            writer.append(':')
-            writer.append(dependency.version)
-            writer.appendln()
+            writer.appendDependency(
+                group = dependency.group,
+                name = dependency.name,
+                version = dependency.version
+            )
         }
     }
 }
 
+internal fun Settings.clearUsedPlugins() {
```

Comment:
naming also not really clear here

Reply:
Renamed to `clearUsedPluginsList`. If you have a better idea, please let me know.